pkg/kyma/core/options.go

```diff
@@ -0,0 +1,18 @@
+package core
```

Comment:
I didn't knew where to place it better to be usable across all components, feel free to move it

Reply:
Why separate package is needed in your opinion? ' so tightly coupled with `kyma` command that I would see it in `kyma` package.

Reply:
Not sure what you mean with "kyma" package, can you point me to a folder